### PR TITLE
Spring20 robot script fixes WIP

### PR DESCRIPTION
**Pull request by skristem**
fixing a test case in affiliations and also contacts 
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: robot/EDA/resources/ContactsPageObject.py [hunk 1]
```diff
@@ -91,17 +110,18 @@ def select_contact(self,contact_firstname, contact_lastname):
...
+#         self.selenium.wait_until_page_contains_element(
+#             contacts_locators["edit_contact"], 
+#             timeout=60,
+#             error="Edit button not found for Contact"
+#         )
+#         self.open_item(
+#             contacts_locators["edit_contact"], 
+#             "Edit button not found for Contact", 
+#             False
+#         )
```
Comment (sgovindankutty) [inline robot/EDA/resources/ContactsPageObject.py]: Can this be removed if not required?

File: robot/EDA/resources/ContactsPageObject.py [hunk 2]
```diff
@@ -91,17 +110,18 @@ def select_contact(self,contact_firstname, contact_lastname):
...
+        self.salesforce.click_object_button("Edit")
+        self.salesforce.wait_until_modal_is_open()
         self.selenium.driver.execute_script(
             "arguments[0].scrollIntoView()", 
             self.selenium.driver.find_element_by_xpath(contacts_locators["preferred_phone"])
```
Comment (sgovindankutty) [inline robot/EDA/resources/ContactsPageObject.py]: consider assigning to a variable and calling the variable?

File: robot/EDA/resources/ContactsPageObject.py [hunk 3]
```diff
@@ -378,22 +402,25 @@ def Add_home_phone_to_contact_and_verify(self, FirstName, LastName):
...
+#         self.open_item(
+#             contacts_locators["details_tab"], 
+#             "Details tab not found on contact", 
+#             True
+#         )
+#         self.open_item(
+#             contacts_locators["edit_contact"], 
+#             "Edit button not found on contact", 
+#             False
+#         )
```
Comment (sgovindankutty) [inline robot/EDA/resources/ContactsPageObject.py]: Same as above comment on commenting

File: robot/EDA/resources/EDA.py [hunk 4]
```diff
@@ -481,26 +483,51 @@ def choose_panel_tab(self,tab):
...
+            if self.check_if_element_exists(locator):
+                print(locator)
+                buttons = self.selenium.get_webelements(locator)
+                for button in buttons:
+                    print(button)
+                    if button.is_displayed():
+                        print("button displayed is {}".format(button))
+                        self.salesforce._focus(button)
+                        button.click()
+                        time.sleep(5)
```
Comment (sgovindankutty) [inline robot/EDA/resources/EDA.py]: Is this waiting for something to load?

Comment (skristem) [reply robot/EDA/resources/EDA.py]: yes as I explained to you in the meeting, not putting it here

File: robot/EDA/resources/ContactsPageObject.py (hunk 2 above)
Comment (skristem) [reply robot/EDA/resources/ContactsPageObject.py]: Not making much changes to existing code (only simple needed stuff)as it will all get rewritten later

**Pull request merged**